Makefile

```diff
@@ -505,6 +505,19 @@ ifeq ($(PCH), 1)
   endif
 endif
 
+# Workaround for GCC 7 and 8.
+FS_TEST_FILE := $$(printf "\#include <filesystem>\\\n int main() { std::filesystem::path p; p.parent_path(); }")
+$(shell echo "Debug: CXX='$(CXX)'" 1>&2)
+$(shell echo "Debug: FS_TEST_FILE='$(FS_TEST_FILE)'" 1>&2)
+#ifeq   ($(shell echo "$(FS_TEST_FILE)" | $(CXX) -o /dev/null -x c++ -std=c++17 -            >/dev/null 2>&1 || echo fail),fail)
+  #ifeq ($(shell echo "$(FS_TEST_FILE)" | $(CXX) -o /dev/null -x c++ -std=c++17 - -lstdc++fs >/dev/null 2>&1 || echo fail),fail)
+ifeq   ($(shell echo "$(FS_TEST_FILE)" | $(CXX) -o /dev/null -x c++ -std=c++17 -            || echo fail),fail)
+  ifeq ($(shell echo "$(FS_TEST_FILE)" | $(CXX) -o /dev/null -x c++ -std=c++17 - -lstdc++fs || echo fail),fail)
+    $(error "Unable to compile test filesystem program")
+  endif
+  LDFLAGS += -lstdc++fs
```

Comment:
Maybe you can simply specify this flag when invoking `make` in your local workflow.